cmd/create/oidcconfig/cmd.go

```diff
@@ -390,6 +399,18 @@ func (s *CreateOidcConfigManualStrategy) execute(r *rosa.Runtime) {
 		Build()
 	commands = append(commands, putJwksCommand)
 	commands = append(commands, fmt.Sprintf("rm %s", jwksFilename))
+	createSecretCommand := awscb.NewSecretsManagerCommandBuilder().
```

Comment:
Changing so that if the bucket name is still not valid for some reason we don't create the secret